clusterloader2/testing/watch-list/config.yaml

```diff
@@ -0,0 +1,71 @@
+{{$testDuration := "5m"}}
+name: huge-secrets
+namespace:
+  number: 1
+  prefix: "huge-secrets"
+tuningSets:
+- name: Uniform10qps
+  qpsLoad:
+    qps: 10
+steps:
+- name: Create secrets
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+      basename: huge-secrets
+    replicasPerNamespace: 400
+    tuningSet: Uniform10qps
+    objectBundle:
+    - basename: huge-secret
+      objectTemplatePath: "secret.yaml"
+- name: Start measurements
+  measurements:
+    - Identifier: TestMetrics
+      Method: TestMetrics
+      Params:
+        action: start
+    - Identifier: WaitForControlledPodsRunning
+      Method: WaitForControlledPodsRunning
+      Params:
+        action: start
+        apiVersion: apps/v1
+        kind: Deployment
+        labelSelector: group = watch-list
+        operationTimeout: 60s
+- name: Start the secret informer
+  phases:
+    - namespaceRange:
+        min: 1
+        max: 1
+        basename: huge-secrets
+      replicasPerNamespace: 1
+      tuningSet: Uniform10qps
+      objectBundle:
+        - basename: huge-secret
+          objectTemplatePath: role.yaml
+        - basename: huge-secret
+          objectTemplatePath: roleBinding.yaml
+        - basename: huge-secret
+          objectTemplatePath: "deployment.yaml"
+          templateFillMap:
+            Replicas: 1
+            Duration: {{$testDuration}}
```

Comment:
What does it give us? Even if the binary will finish, the pod itself will get restarted (the default restart policy is Always), no?

Also - by definition deployment is for long-running stuff. Should we instead of `Job` here?

Reply:
> What does it give us? Even if the binary will finish, the pod itself will get restarted (the default restart policy is Always), no?

yes, that is correct.

> Also - by definition deployment is for long-running stuff. Should we instead of Job here?

I don't have a preference. We could switch to a Job but we need to define when our test ends.

I think we would need `testDuration` anyway even for a job, do you agree?

Reply:
We would need testDuration - but that would only be an argument to the binary itself.

And then test finishes when job finishes.

Reply:
yes, okay let me rework the manifest, thanks.

Reply:
okay, switched to a Job, PTAL.